packages/core/phase1/lib/errorPaths.ts

```diff
@@ -5,13 +5,23 @@ const offencePath = (offenceIndex: number): ExceptionPath => [
   "HearingOutcome",
   "Case",
   "HearingDefendant",
-  "Offence",
-  offenceIndex
+  ...(offenceIndex < 0 ? [] : ["Offence", offenceIndex])
```

Comment:
When offence index is -1, it means Result is not associated with and offence. (HearingDefendant > Result)